templates/common/on-prem/units/nodeip-configuration.service.yaml

```
@@ -26,7 +26,7 @@ contents: |
     {{ .Images.baremetalRuntimeCfgImage }} \
     node-ip \
     set --retry-on-failure \
-    {{ onPremPlatformAPIServerInternalIP . }}; \
+    {{- range onPremPlatformAPIServerInternalIPs . }}{{.}} {{end}}; \
```

@cybertron is it intended, to suppress the newline here?

---

Clearly not, that's the error, and what is causing the IP to be prefix with the `\` char.
I suggest you also add quotes around the IP address:
```suggestion
    {{range onPremPlatformAPIServerInternalIPs . }}"{{.}}" {{end}}; \
```

---

Huh, I wonder how that ever worked. I wonder if we just got lucky and the default kubelet behavior was working even though nodeip-configuration failed? In any case, I've removed that and pushed a new version. Let's see what ci says.